### PR TITLE
Fixes podman and docker issue

### PR DESCRIPTION
**Pull request by udaysk23**
Fixes podman and docker issue

### CODE REVIEW

File: tests/sanity_rgw.py [hunk 1]
```diff
@@ -33,28 +40,98 @@ def run(ceph_cluster, **kw):
...
+        # rgw_ceph_object.exec_command(cmd='sudo rm -rf ' + test_folder)
+        # rgw_ceph_object.exec_command(cmd='sudo mkdir ' + test_folder)
+        test_cmd = 'python3 ' + test_folder_path + script_dir + script_name + ' -c ' \
+                   + test_folder + config_dir + config_file_name
+        build_a = ceph_cluster.rhcs_version
+        build = str(build_a)
+        distro_info = rgw_node.distro_info
+        distro_ver = distro_info['VERSION_ID']
+        if build.startswith('4'):
+            if distro_ver.startswith('8'):
```
Comment (rakeshgm) [inline tests/sanity_rgw.py:52]: There is a lot of repetitive code here. please check again. there is definitely room for improvement

Comment (XDsubhash) [reply tests/sanity_rgw.py:52]: @udaysk23 any update on this?

**Comment by udaysk23** [comment]
Logs are available in following location
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1592986644575/

File: tests/sanity_rgw.py (hunk 1 above)
Comment (rakeshgm) [reply tests/sanity_rgw.py:79]: I still fee there is a room for reducing all these `if`s. but for now It's okay, maybe we can revist again later

**Comment by rakeshgm** [comment]
This PR address this issue [128](https://github.com/red-hat-storage/cephci/issues/128)

**Pull request merged**